client/src/components/Cart/CartItem.js

```diff
@@ -0,0 +1,52 @@
+import { useData } from '../../context/DataContext';
+import { RemoveCartItemButton } from '../Buttons/RemoveCartItemButton';
+import { MoveToWishListButton } from '../Buttons/MoveToWishListButton';
+import './CartItem.css';
+
+export const CartItem = ({ product }) => {
+   const { dispatch } = useData();
+
+   const handleIncrementQty = (product) => {
+      return dispatch({ type: 'INC_QTY', payload: product });

```

Comment:
for dispatch type you can use full names like INCREASE_QUANTITY.... i know it is sort of nitpicking the code but would be much easier to read code afterwards!

Reply:
No, you're right! I should be as verbose as possible while naming dispatch.
Thanks for pointing it out!